sections/exchange/FooterCard/common.tsx

```diff
@@ -49,13 +49,25 @@ export const SummaryItemValue = styled.div`
 `;
 
 export const MessageContainer = styled(GridDivCentered)<{ attached?: boolean }>`
+	display: grid;
+	-webkit-box-align: center;
```

Comment:
`-webkit` probably not needed. Also I'm not sure we should be using this? Looks pretty old school. Thoughts?

Reply:
fixed.